iOS/Plugins/FlipperKitNetworkPlugin/FlipperKitNetworkPlugin/SKResponseInfo.m

```diff
@@ -29,6 +29,11 @@ - (instancetype)initWithIndentifier:(int64_t)identifier
 }
 
 + (BOOL)shouldStripReponseBodyWithResponse:(NSURLResponse*)response {
+  // Only HTTP(S) responses have Content-Type headers
+  if (![response isKindOfClass:[NSHTTPURLResponse class]]) {
```

Comment:
I'm wondering it this shouldn't return `YES` instead, given that this method returns if the response data should be stripped from the network request details, which is the case for large binary data as images, videos and zips. I can imagine we don't want to send the details fo `data:` urls to Flipper either? (see also line 50)

Reply:
That makes sense. I updated this PR.